resources/server/src/landingpage/README.md

```diff
@@ -1,6 +1,6 @@
 # Landing page/catalog webapp
 
-[Landing page/catalog webapp](https://docs.qgis.org/latest/en/docs/server_manual/services.html#qgis-server-catalog) source code.
+[Landing page/catalog webapp](https://docs.qgis.org/3.34/en/docs/server_manual/catalog.html) source code.
```

Comment:
Shouldn't this still be "latest" rather then the hardcoded version?

(Ping @DelazJ )

Reply:
Yes, better keep latest.

Reply:
thanks for catching that :100: